graylog2-bootstrap/src/main/resources/log4j.xml

```diff
@@ -44,6 +44,9 @@
     <logger name="kafka.log.OffsetIndex">
         <level value="warn"/>
     </logger>
+    <logger name="org.graylog2.security">
```

Comment:
The logger `org.graylog2`, which is a superset of `org.graylog2.security`, is already configured to log on level INFO, so this is not necessary.